roles/kibana/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+kibana_package_name: kibana
+kibana_service_name: kibana
+kibana_config_dir: /etc/kibana
+kibana_config_file: '{{ kibana_config_dir }}/kibana.yml'
+kibana_config_mode: 0644
+kibana_owner: kibana
+kibana_group: kibana
+
+# "0.0.0.0" to allow public access
+# "localhost" to allow access through proxy only
+kibana_server_bind: localhost
+kibana_server_port: 5601
+kibana_server_elasticsearch_url: 'http://{{ elasticsearch_bind }}:9200'
```

Comment:
Is this the same `elasticsearch_bind` that is used to specify which address elasticsearch listens to - https://github.com/centos-opstools/opstools-ansible/pull/14/commits/f9f7b5047a477c004918b6e6a6c91f800ae024e5 ?

If so, this needs to be a different value - `elasticsearch_host` or `elasticsearch_addr` or something like that, to differentiate this from the elasticsearch server bind address.  I realize that a user could just pass in `kibana_server_elasticsearch_url`.  Do we need some sort of elasticsearch client role, which will be used for fluentd and kibana?  All elasticsearch clients will need to be able to specify es host, port, scheme (http or https), username, password, cert, key, ca cert, possibly other options as well.